app/src/lib/git/status.ts

```diff
@@ -256,6 +257,15 @@ function getConflictStatus(
     return { kind: 'text', conflictMarkerCount }
   }
 
+  const uncoveredConflict =
```

Comment:
:art: I'm trying to think of a more descriptive name for this parameter that fits with how it's used downstream but nothing really jumps out - `conflictRequiresManualResolution` feels too wordy, `conflictWithoutMarkers` maybe? How attached are you to `uncovered` here?

Reply:
not attached at all! i've been trying to think of a better name, too. i like `conflictsWithoutMarkers`.